src/style.css

```diff
@@ -1 +1,67 @@
-/* add styles here */
+:root {
+  --main-color: #e1e7ed;
+  --text-color: #3d4e61;
+  --shadow: rgba(0, 0, 0, 0.25);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 600;
+}
+
+* {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+  box-sizing: border-box;
```

Comment:
Don't use `*` selector as it impacts performance. It's better to set styles only for elements that require them. If you need to apply a global font, consider applying it to the `body` element instead.